packages/db-ivm/src/operators/topKWithFractionalIndex.ts

```diff
@@ -261,30 +258,28 @@ export class TopKWithFractionalIndexOperator<K, T> extends UnaryOperator<
     super(id, inputA, output)
     const limit = options.limit ?? Infinity
     const offset = options.offset ?? 0
-    const compareTaggedValues = (
-      a: TaggedValue<K, T>,
-      b: TaggedValue<K, T>
-    ) => {
+    const compareKeyedValues = ([aKey, aVal]: [K, T], [bKey, bVal]: [K, T]) => {
```

Comment:
nit: I would extract this to the module-level instead of nesting this definition in the constructor.